packages/framework/src/Resources/scripts/admin/components/autocompleteSelectbox.js

```diff
@@ -2,6 +2,38 @@
 
     Shopsys = window.Shopsys || {};
 
+    // TODO: till this https://github.com/selectize/selectize.js/issues/1395 issue is not fixed
+    Selectize.prototype.positionDropdown = function () {
+        var $control = this.$control;
+        var offset = this.settings.dropdownParent === 'body' ? $control.offset() : $control.position();
+        var bottomOffset = $('.js-window-fixed-bar').height() || 0;
+        offset.top += $control.outerHeight(true);
+
+        var css = {
+            width: $control.outerWidth(),
+            top: offset.top,
+            left: offset.left,
+            bottom: 'auto'
+        };
+
+        this.$dropdown.css(css);
+
+        if (this.$dropdown.offset().top + this.$dropdown.height() - $(window).scrollTop() + bottomOffset > $(window).height()) {
+            css.bottom = offset.top;
+            css.top = 'auto';
+            this.$dropdown.css(css);
+        }
+
+    };
+
+    // extending of prototype
```

Comment:
is this necessary? I do not see any added value of the comment - please remove it or make it more descriptive